test/e2e/util.go

```diff
@@ -645,7 +646,7 @@ func waitJobStateAborted(ctx *context, job *vkv1.Job) error {
 
 func waitJobPhaseExpect(ctx *context, job *vkv1.Job, state vkv1.JobPhase) error {
 	var additionalError error
-	err := wait.Poll(100*time.Millisecond, oneMinute, func() (bool, error) {
+	err := wait.Poll(100*time.Millisecond, twoMinute, func() (bool, error) {
```

Comment:
this may increase total test time, I would like to add a parameter to `waitJobPhaseExpect`

Reply:
Updated